src/core_io.h

```diff
@@ -44,8 +44,8 @@ UniValue ValueFromAmount(const CAmount amount);
 std::string FormatScript(const CScript& script);
 std::string EncodeHexTx(const CTransaction& tx, const int serializeFlags = 0);
 std::string SighashToStr(unsigned char sighash_type);
-void ScriptPubKeyToUniv(const CScript& scriptPubKey, UniValue& out, bool fIncludeHex, bool include_addresses);
-void ScriptToUniv(const CScript& script, UniValue& out, bool include_address);
-void TxToUniv(const CTransaction& tx, const uint256& hashBlock, bool include_addresses, UniValue& entry, bool include_hex = true, int serialize_flags = 0, const CTxUndo* txundo = nullptr);
+void ScriptPubKeyToUniv(const CScript& scriptPubKey, UniValue& out, bool include_hex, bool include_address = true);
+inline void ScriptToUniv(const CScript& script, UniValue& out) { ScriptPubKeyToUniv(script, out, /* include_hex */ true, /* include_address */ false); }
```

Comment:
6604cdcccd maybe I'm missing something but perhaps merge `ScriptPubKeyToUniv` and `ScriptToUniv` into a single function?